blockly-core/core/utils/xml.js

```diff
@@ -17,6 +17,12 @@
  * limitations under the License.
  */
 
+/* global Blockly, goog */
+
+// Ignore "Expected a conditional expression and instead saw an
+// assignment" warnings.
+/* jshint -W084 */
```

Comment:
Agreed. adding an ignore for this one style change at the top of the file was intended to be a small step toward (b).